### PR TITLE
executor: Update admin show ddl test

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: executor/executor_test.go [hunk 1]
```diff
@@ -63,12 +64,16 @@ func (s *testSuite) TestAdmin(c *C) {
 	ddlInfo, err := inspectkv.GetDDLInfo(txn)
 	c.Assert(err, IsNil)
 	c.Assert(row.Data[0], Equals, ddlInfo.SchemaVer)
-	c.Assert(row.Data[1], DeepEquals, ddlInfo.Owner.String())
+	rowOwnerInfos := strings.Split(row.Data[1].(string), ",")
+	ownerInfos := strings.Split(ddlInfo.Owner.String(), ",")
+	c.Assert(rowOwnerInfos[0], DeepEquals, ownerInfos[0])
```
Comment (shenli) [inline executor/executor_test.go]: DeepEquals->Equals?

**Comment by zimulala** [comment]
PTAL @shenli @coocood

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**